### ISSUE TITLE
Erros while using cases_small

### ISSUE THREAD

**Issue by PriyankaKushwaha940** [NREL/ReEDS-2.0#163]
Hi,
while running the runbatch.py for cases_small [ReEDS-2.0](https://github.com/NREL/ReEDS-2.0). The script/model is throwing division by zero error (0), no chnages has been done to input ot to the scripts. I am fairly new to the tool, and unable to find the reason for the same.  Some output files (csv) are there but report folder is not generated in output folder. Attached is the 
[gamslog.txt](https://github.com/NREL/ReEDS-2.0/files/13271585/gamslog.txt)
Gamlog for reference.

**Comment by wesleyjcole** [comment]
From the gamslog.txt file it looks like the model ran fine, but threw an error in the e_report.gms script when it was writing the outputs.

I just tried running the head of main on my machine using cases_small, and didn't get the divide by zero error.  Would you check to make sure you are using the head of main?  If you are, then would you send me the report_*.lst file in the lstfiles folder so I can take a look at that?

**Comment by PriyankaKushwaha940** [comment]
Hi Wesley,
Thanks for the responding. 
I tried running the script using head of main as suggested, still the issue persists. Attached is the report file for reference. 
[report_TT1_ercot_small.txt](https://github.com/NREL/ReEDS-2.0/files/13276920/report_TT1_ercot_small.txt)

**Comment by wesleyjcole** [comment]
Have you changed anything in the repository?  I'm not seeing anything that should cause an issue, and am unable to reproduce the error on my own system.

To get the model to produce outputs, you can simply comment out these lines in e_report.gms:

```
curt_rate_tech(i,rb,t)$[rfeas(rb)$tmodel_new(t)$vre(i)$curt_tech(i,rb,t)$gen_out_ann(i,rb,t)] =
    curt_tech(i,rb,t) / (gen_out_ann(i,rb,t) + curt_tech(i,rb,t))
;
```

**Comment by PriyankaKushwaha940** [comment]
Hi,
I tried running the model by commenting the mentioned set of lines. Attaching is the Gams log file for the same. It shows some error like 

KeyError: 'load_category'
[Errno 2] No such file or directory: 'C:\\ReEDS-2.0\\runs\\ThT1_ercot_small\\outputs\\df_sc_out_upv_reduced.csv'
[Errno 2] No such file or directory: 'C:\\ReEDS-2.0\\runs\\ThT1_ercot_small\\outputs\\df_sc_out_wind-ons_reduced.csv'
[Errno 2] No such file or directory: 'C:\\ReEDS-2.0\\runs\\ThT1_ercot_small\\outputs\\df_sc_out_wind-ofs_reduced.csv'
[Errno 2] No such file or directory: 'C:\\ReEDS-2.0\\runs\\ThT1_ercot_small\\outputs\\df_sc_out_upv_reduced.csv'
[Errno 2] No such file or directory: 'C:\\ReEDS-2.0\\runs\\ThT1_ercot_small\\outputs\\df_sc_out_wind-ons_reduced.csv'
[Errno 2] No such file or directory: 'C:\\ReEDS-2.0\\runs\\ThT1_ercot_small\\outputs\\df_sc_out_wind-ofs_reduced.csv'
map-VREsites-translines failed:
'DataFrame' object has no attribute 'Dim3'
map_agg failed:
'NorthernGrid'

Will these impact the model output. Or shall these can be ignored and output can be analyzed. I have attached log file for reference. 
[Report.txt](https://github.com/NREL/ReEDS-2.0/files/13304358/Report.txt)


Thanks

**Comment by wesleyjcole** [comment]
I randomly stubbled upon this error in our internal version of ReEDS, which let me make a fix for this issue.  The fix was pushed to main in this commit: 6ff0cc7eac80e24ee3bf435470add3d485090d99.
